routers/routers.go

```diff
@@ -27,28 +45,21 @@ func DefineRoutes() http.Handler {
 
 	// Redirects
 	mux.HandleFunc("/", handlers.RedirectToPathHandler(paths.HomePage))
+	mux.HandleFunc("/api/", http.NotFound)
```

Comment:
without this line. /api/bobby will redirect to /home. Which can be confusing if your api calls returns a redirect, then resolves to the home page. Now for bad invalid api calls it just returns 404.

Reply:
this functions as similar to regex (and "executed" in this order)
/api/* will return a 404 on error
/* will return a redirect on error

Reply:
hmm okay. I guess I'm confused about what we're going for? I feel like there's a lot of different behaviors here. Is there a reason we don't just return a 404 for all endpoints we have yet to define? Wouldn't that be the most correct?

Reply:
We could if you wanted to. The redirect all bad pages was a side-effect (i was pleasantly surprised with ) of adding a redirect for just /.

Reply:
we could. I personally like the redirect for bad pages, but i could see it being frustrating for users.  

I also like having the control of behavior between api endpoints and web pages. The not-authenticated is a really good example. Before if you tried an api call and you forgot to included your cookie you'd get a temporary redirect to the login page. Now it warns you that you that you need a better bearer, including a message of where it failed in the process.